tokio/src/signal/registry.rs

```diff
@@ -202,7 +202,9 @@ mod tests {
                 registry.broadcast();
 
                 // Yield so the previous broadcast can get received
-                crate::time::sleep(std::time::Duration::from_millis(10)).await;
+                for _ in 0..100 {
```

Comment:
nit, tioli: maybe worth a comment that the reason we do this with a bunch of yields in a loop is for miri? not a blocker...

Reply:
I mean, it's good for non-miri too since it makes the test faster.